pkg/constants/constants.go

```diff
@@ -43,6 +43,12 @@ var (
 	KFServiceGKEAcceleratorAnnotationKey = KFServingAPIGroupName + "/gke-accelerator"
 )
 
+// KFService Internal Annotations
+var (
+	KFServicePrivateAnnotationsPrefix                       = KFServingAPIGroupName + "/internal"
+	KFServiceModelInitializerSourceURIInternalAnnotationKey = KFServicePrivateAnnotationsPrefix + "/model-initializer/sourceURI"
```

Comment:
This is a variable name given by the gods.

Reply:
LOL!... suggestions welcome :)